File: rb/utility/make_context.rb
# NovayaGazeta SDK utility: make_context
require_relative '../core/context'
module NovayaGazetaUtilities
  MakeContext = ->(ctxmap, basectx) {
    NovayaGazetaContext.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// NovayaGazeta SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class NovayaGazetaMakeContext
{
    public static function call(array $ctxmap, ?NovayaGazetaContext $basectx): NovayaGazetaContext
    {
        return new NovayaGazetaContext($ctxmap, $basectx);
    }
}
